python/lsst/rapid/analysis/focusAnalysis.py

```diff
@@ -56,9 +55,9 @@ class FitResult:
 
 class SpectralFocusAnalyzer():
 
```

Comment:
As was brought up before: docstrings for the class (and for `NonSpectralFocusAnalyzer` below).

Reply:
And for the class methods.